File: BEGINNER/1046 Tempo de Jogo.py
#algoritmo
# RECEBER O DOIS HORÁRIOS(HORA INICIAL E HORA FINAL)
# DETERMINAR A HORA QUE PASSOU
# SE HI > HF, (HI - HF + 24)
# SE HI == HF, (24)
# SE HI < HF, (HF - HI)

# PROGRAMA PRINCIPAL
Hi, Hf = map(int, input().split())
if Hi > Hf:
    resultado = ((Hf + 24) - Hi)
elif Hi < Hf:
    resultado = (Hf - Hi)
else:
    resultado = 24
print('O JOGO DUROU %d HORA(S)' % resultado)

File: C/INICIANTE/1046 Tempo de Jogo.c
#include <stdio.h>
 
int main() 
{
    int hi, hf;
    scanf("%d %d", &hi, &hf);
    if (hi == hf)
        printf("O JOGO DUROU 24 HORA(S)\n");
    else if (hi > hf)
        printf("O JOGO DUROU %d HORA(S)\n", (24 - (hi - hf)));
    else
        printf("O JOGO DUROU %d HORA(S)\n", (hf - hi));
    return 0;
}
